x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_form.test.tsx

```diff
@@ -164,20 +167,14 @@ describe('action_form', () => {
           application: { capabilities },
         },
       ] = await mocks.getStartServices();
-      deps = {
-        toastNotifications: mocks.notifications.toasts,
-        http: mocks.http,
-        capabilities: {
-          ...capabilities,
-          actions: {
-            delete: true,
-            save: true,
-            show: true,
-          },
+      // eslint-disable-next-line react-hooks/rules-of-hooks
+      useKibanaMock().services.application.capabilities = {
```

Comment:
If there's a way that avoids mutating the result of `useKibanaMock()`, I'm +1 to explore that but I'm not sure of any alternatives at this time unless passed into `useKibanaMock(...here...)`?